### DISCUSSION TITLE
How to delete models concurrently

### DISCUSSION THREAD

**Discussion by xiazhibin** [python-gino/gino#789] [Q&A]
because of `self.acquire(reuse=True)`, I have to 
```
await Model.delete.where(id==1).gino.status()
await Model.delete.where(id==2).gino.status()
```

I cann't use `asyncio.gather(Model.delete.where(id==1).gino.status(), Model.delete.where(id==2).gino.status())`, it will raise exception `asyncpg.exceptions._base.InterfaceError: cannot perform operation: another operation is in progress`.

Here is the issuse #313 

So how can I delete models concurrently

**Comment by worlak2** [comment]
Maybe try transaction and create task?
